### PR TITLE
Update Remote execution spec to build for el8

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -1,15 +1,18 @@
 # template: foreman_plugin
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
+%{!?_root_sbindir:%global _root_sbindir %{_sbindir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
 
+%global debug_package %{nil}
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec:7]: For my understanding: why is this needed?

**Comment by zjhuntin** [comment]
[test rpm]

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec (hunk 1 above)
Comment (evgeni) [reply packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec:7]: `noarch` packages should not produce any debug symbols, so it should not be needed. but with el8 you never know.
